db/schema.rb

```diff
@@ -44,6 +54,7 @@
     t.string   "last_sign_in_ip"
     t.datetime "created_at"
     t.datetime "updated_at"
+    t.string   "name"
```

Comment:
LGTM. Not the best with devise + omniauth but I looked at some other examples and this largely follows what they do. Thanks for doing this :+1: